acceptance/auth/credentials/oauth/out.requests.txt

```diff
@@ -0,0 +1,36 @@
+{
+  "headers": {
+    "User-Agent": [
+      "cli/[DEV_VERSION] databricks-sdk-go/[SDK_VERSION] go/[GO_VERSION] os/[OS]"
+    ]
+  },
+  "method": "GET",
+  "path": "/oidc/.well-known/oauth-authorization-server",
+  "body": ""
```

Comment:
Unrelated to this PR: the `body` field can be `omitempty`.

Reply:
fixed in https://github.com/databricks/cli/pull/2333